kilkari/src/main/java/org/motechproject/nms/kilkari/service/DeactivationAuditService.java

```diff
@@ -0,0 +1,11 @@
+package org.motechproject.nms.kilkari.service;
+
+import org.motechproject.nms.kilkari.domain.SubscriptionOrigin;
+import org.motechproject.nms.kilkari.domain.SubscriptionStatus;
+
+public interface DeactivationAuditService {
+
+    void auditSuccess(String subscriptionId, Long subscriberId, SubscriptionOrigin subscriptionOrigin, SubscriptionStatus status, Long msisdn);
```

Comment:
Followed the existing structure (audit service from csv package). Thanks for the suggestion, will change it accordingly